### PR TITLE
Tweak attribute parsing some more

### PR DESCRIPTION
**Pull request by djc**
I think this a little easier to follow than the result from #603, what do you think?

### CODE REVIEW

File: askama_shared/src/input.rs [hunk 1]
```diff
@@ -36,16 +36,15 @@ impl TemplateInput<'_> {
...
-                        return Err(CompileError::Static(
-                            "attribute 'template' has incorrect type",
-                        ));
+
+                match attr.parse_meta() {
+                    Ok(syn::Meta::List(syn::MetaList { nested, .. })) => {
+                        template_args = Some(nested);
+                    }
+                    Ok(_) => {
+                        return Err("'template' attribute must be a list".into());
```
Comment (vallentin) [inline askama_shared/src/input.rs:45]:
```suggestion
                    Ok(_) => return Err("'template' attribute must be a list".into()),
```

To match the following one. Unless it was intentional? :)

**Comment by Kijewski** [comment]
I like it! It's easier to follow than my variant.

File: askama_shared/src/input.rs (hunk 1 above)
Comment (Kijewski) [reply askama_shared/src/input.rs:45]: :+1:

File: askama_shared/src/input.rs [hunk 2]
```diff
@@ -36,16 +36,14 @@ impl TemplateInput<'_> {
...
-                        return Err(CompileError::Static(
-                            "attribute 'template' has incorrect type",
-                        ));
+
+                match attr.parse_meta() {
+                    Ok(syn::Meta::List(syn::MetaList { nested, .. })) => {
+                        template_args = Some(nested);
+                    }
+                    Ok(_) => return Err("'template' attribute must be a list".into()),
                     }
```
Comment (vallentin) [inline askama_shared/src/input.rs:45]: This one needs to be removed as well. But couldn't mark it, while making the initial suggestion

**Review by Kijewski — APPROVED** [review]

**Pull request merged**